src/wrappers/themis/Obj-C/objcthemis/objcthemis.h

```diff
@@ -1,14 +1,14 @@
 #ifndef _OBJCTHEMIS_
     #define _OBJCTHEMIS_
 
-	#import <objcthemis/scell_seal.h>
-	#import <objcthemis/scell_token.h>
-	#import <objcthemis/scell_context_imprint.h>
-	#import <objcthemis/skeygen.h>
-	#import <objcthemis/smessage.h>
-	#import <objcthemis/scomparator.h>
-	#import <objcthemis/ssession.h>
-	#import <objcthemis/serror.h>
+    #import "scell_seal.h"
+    #import "scell_token.h"
+    #import "scell_context_imprint.h"
+    #import "skeygen.h"
+    #import "smessage.h"
+    #import "scomparator.h"
+    #import "ssession.h"
+    #import "serror.h"
```

Comment:
I'm a bit confused wrt the usage of `<objcthemis/serror.h>` vs `"serror.h"`. Shouldn't it be backwards instead?

- `<objcthemis/serror.h>` should be used in header files as that's what included into the user code
- `"serror.h"` should be used in source files to ensure that we compile with headers that are nearby

Reply:
Are you confused with any of this change `<objcthemis/smessage.h>` vs `"smessage.h"` or only with `serror.h`?
Because for me it's also a bit weird: this thing is working properly for ObjC, but these headers are "non-modulars" for Swift frameworks.

Reply:
All of them, not just `serror.h`.

I have a feeling that we have to use `"serror.h"` style here because we copy the headers as a flat list when constructing the framework. I remember seeing in Xcode that all of the files are just dumped into the framework source without preserving the subdirectories. Though, we seem to use this 'subspec' thing of Cocoapods which should group files properly...

Reply:
yes, somehow Xcode/Cocoapods don't pay attention on subdirectories, and "umbrella" header also consists of "flat" headers ¯\_(ツ)_/¯ 
well, that's what we have here..